implementations/micrometer-registry-elastic/src/main/java/io/micrometer/elastic/ElasticMeterRegistry.java

```diff
@@ -287,11 +286,21 @@ static int countCreatedItems(String responseBody) {
     }
 
     /**
-     * Return index name.
-     *
-     * @return index name.
-     * @since 1.2.0
-     */
+	 * Return index type. default is 'doc'
```

Comment:
We should note here that this only applies to Elasticsearch versions before 7.

Reply:
Thanks, done.